frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx

```diff
@@ -10,29 +10,52 @@ import {
   ChartThemeColor,
   ChartTooltip,
 } from '@patternfly/react-charts';
+import { Tooltip } from '@patternfly/react-core';
 import { DataPoint } from '@console/internal/components/graphs';
-import { K8sKind } from '@console/internal/module/k8s';
+import { K8sKind, referenceForModel } from '@console/internal/module/k8s';
 import { resourcePathFromModel } from '@console/internal/components/utils';
+import { CEPH_STORAGE_NAMESPACE } from '@console/ceph-storage-plugin/src/constants';
 import { isAvailableBar, getBarRadius, StackDataPoint } from './utils';
 import './breakdown-card.scss';
 
-const LinkableLegend: React.FC<LinkableLegendProps> = (props: LinkableLegendProps) => {
-  const { metricModel, datum } = props;
-  const href = metricModel
-    ? resourcePathFromModel(metricModel, datum.link)
-    : '/dashboards/object-service';
-  return (
-    <Link to={href} className="capacity-breakdown-card__legend-link">
+const LinkableLegend: React.FC<LinkableLegendProps> = React.memo((props: LinkableLegendProps) => {
+  const { metricModel, datum, ocsVersion } = props;
+  let href: string = resourcePathFromModel(metricModel, datum.link, CEPH_STORAGE_NAMESPACE);
+  const style = {};
+  const customLegend = (
+    <Tooltip content={datum.link} enableFlip>
       <ChartLabel
         {...props}
         lineHeight={1.2}
         style={[{ ...datum.labels, fontSize: 9 }, { fill: 'black', fontSize: 8 }]}
       />
+    </Tooltip>
+  );
+  if (datum.name[0] === 'Others' || datum.name[0] === 'Cluster-wide') {
```

Comment:
All the strings should be moved to constant file, so that if it gets changed later, it will be easy to do that.

Reply:
This will be used only here.
I can make consts though.